cookbook/providers/openai/data_analyst.py

```diff
@@ -0,0 +1,19 @@
+from phi.agent import Agent
+from phi.model.openai import OpenAIChat
+from phi.tools.duckdb import DuckDbTools
+
+duckdb_tools = DuckDbTools(create_tables=False, export_tables=False, summarize_tables=False)
+duckdb_tools.create_table_from_path(
+    path="https://phidata-public.s3.amazonaws.com/demo_data/IMDB-Movie-Data.csv", table="movies"
+)
+
+agent = Agent(
+    model=OpenAIChat(model="gpt-4o"),
+    tools=[duckdb_tools],
+    show_tool_calls=True,
+    add_to_system_prompt="""
```

Comment:
@jacobweiss2305 @ysolanky what's a better variable for `add_to_system_prompt` -- it seems clear but a bit too crude. Looking for suggestions that looks neater like: `additional_information`, `additional_context`

I do like `additional_context` what do you think?

Reply:
+1 additional_context

Reply:
This should be PR for agent class. happy to make commit once we all agree